website/docs/r/aml_file_system.html.markdown

```diff
@@ -0,0 +1,138 @@
+---
+subcategory: "AML File System"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_aml_file_system"
+description: |-
+  Manages an AML File System.
```

Comment:
Can we expand the abbr. name `AML` to its full name among this document?

Reply:
Updated